core/EditingTools/gridAndLabelCreator.py

```diff
@@ -1,425 +1,716 @@
+			+','+str('\' \'')+',round((round((-floor(round(abs('+str(x)+'),4))+round(abs('+str(x)+'),4)),4)*60-floor(round((-floor(round(abs('+str(x)+'),4))+round(abs('+str(x)+'),4)),4)*60))*60),'+str('\'\'\'\'')+')),if('+str(x) \
+			+'<0, '+str('\' ')+neg_character+str('\'')+','+str('\' ')+pos_character+str('\'')+'))'
+		
+		return conv_exp_str
+
+
+
+	def styleCreator (layer, index, spacing, crossX, crossY, scale, color, fontSize, font):
+		
+		grid_spacing = spacing
+		geo_number_x = crossX
+		geo_number_y = crossY
+		map_scale = scale*1000
+		grid_color = color
+		fSize = fontSize
+		fontType = font
+		
+		#Loading feature
+		layer_bound = layer
+		feature_bound = layer_bound.getFeature(0)
+
+		#Getting Feature Source CRS and Geometry
+		bound_sourcecrs = layer_bound.crs().authid()
+		feature_bbox = feature_bound.geometry().boundingBox()
+		geo_bound_bb = str(feature_bbox).replace(',','').replace('>','')
+
+		#Defining CRSs Transformations
+		inom = feature_bound[index]
+		if inom[0]=='N': 
+			bound_UTM = 'EPSG:319' + str(72 + int(inom[3:5])-18)
+		elif inom[0]=='S': 
+			bound_UTM = 'EPSG:319' + str(78 + int(inom[3:5])-18) 
+		else:
+			iface.messageBar().pushMessage("Error", "Invalid index attribute", level=Qgis.Critical)
+			return
+		trLLUTM = QgsCoordinateTransform(QgsCoordinateReferenceSystem(bound_sourcecrs), QgsCoordinateReferenceSystem(bound_UTM), QgsProject.instance())
+		trUTMLL = QgsCoordinateTransform(QgsCoordinateReferenceSystem(bound_UTM), QgsCoordinateReferenceSystem(bound_sourcecrs), QgsProject.instance())
+
+		#Defining UTM Grid Symbology Type
+		renderer = layer_bound.renderer()
+		properties = {'color': 'black'}
+		grid_symb = QgsFillSymbol.createSimple(properties)
+		symb_out = QgsSimpleFillSymbolLayer()
+		symb_out.setStrokeColor(QColor('white'))
+		symb_out.setFillColor(QColor('white'))
+
+
+		""" Creating UTM Grid """
+		geo_UTM = feature_bound.geometry()
+		geo_UTM.transform(trLLUTM)
+		bound_UTM_bb = str(geo_UTM.boundingBox()).replace(',','').replace('>','')
+		xmin_source = float(geo_bound_bb.split()[1])
+		ymin_source = float(geo_bound_bb.split()[2])
+		xmax_source = float(geo_bound_bb.split()[3])
+		ymax_source = float(geo_bound_bb.split()[4])
+		xmin_UTM = float(bound_UTM_bb.split()[1])
+		ymin_UTM = float(bound_UTM_bb.split()[2])
+		xmax_UTM = float(bound_UTM_bb.split()[3])
+		ymax_UTM = float(bound_UTM_bb.split()[4])
+
+		if grid_spacing > 0:
+			UTM_num_x = floor(xmax_UTM/grid_spacing) - floor(xmin_UTM/grid_spacing)
+			UTM_num_y = floor(ymax_UTM/grid_spacing) - floor(ymin_UTM/grid_spacing)
+
+			#Generating Vertical Lines
+			for x in range(1, UTM_num_x+1):
+				grid_symb= GridAndLabelCreator.utm_Symb_Generator (grid_spacing, trUTMLL, trLLUTM, grid_symb, properties, geo_number_x, geo_number_y, UTM_num_x, UTM_num_y, x, 0, xmin_source, xmax_source, ymin_source, ymax_source, xmin_UTM, xmax_UTM, ymin_UTM, ymax_UTM)

```

Comment:
seu self não funciona pois seu super lá em cima está instanciado errado. Só fazer esse tipo de chamada se a classe for estática.